dissect/target/filesystems/extfs.py

```diff
@@ -125,7 +125,8 @@ def lstat(self) -> fsutil.stat_result:
             ]
         )
 
-        # Set the nanosecond resolution separately
+        # Set birthtime and nanosecond resolution separately
+        st_info.st_birthtime = self.entry.crtime.timestamp()
```

Comment:
`self.entry.crtime` can also be `None` and should also be guarded :smile:.

Reply:
Ty, fixed.